src/main/groovy/com/morpheusdata/commvault/CommvaultPlugin.groovy

```diff
@@ -70,4 +73,33 @@ class CommvaultPlugin extends Plugin {
         }
         return backupProvider
     }
+
+    def captureActiveSubclientBackup(authConfig, subclientId, clientId, backupsetId) {
```

Comment:
This appears to be a method that should go in the API service (CommvaultBackupUtility?).

Reply:
@ddeyoung changes has been implemented as suggested.